shoop_tests/core/test_customers.py

```diff
@@ -34,3 +34,17 @@ def test_companies(django_user_model):
             off = (cx * 3 + x) % len(peons)
             contact = get_person_contact(user=peons[off])
             company.members.add(contact)
+
+
+@pytest.mark.django_db
+def test_customer_tax_group1(django_user_model):
+    # test that created person is assigned to proper group
+    person = PersonContact.objects.create(email="test@example.com", name="Test Tester")
+    assert person.tax_group == CustomerTaxGroup.get_default_person_group()
```

Comment:
Add also `assert person.tax_group.identifier == "what_it_is_supposed_to_be"` so that this checks also that `get_default_person_group` returns correct group.